### PR TITLE
Alex colors

### PR DESCRIPTION
**Pull request by AlexLazcano**
Refactored to use styled components in RecentActivity and in ActiveDiscussion component, Implemented theme provider and replaced Global styles with theme.js. I will delete global styles after I make sure it's not used somewhere else. Might be able to place theme provider in a higher component. I separated the routes in the BottomNavBar to be their own files

### CODE REVIEW

File: app/src/components/ProfileComponent/ProfileComponent.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+export default ProfileComponent;
+
+{
+  /* <Text style={styles.isPro}>S|Pro</Text>
+<Text style={styles.name}>Alex Lazcano</Text>
+<Text style={styles.email}>alex.lazcano@gmail.com</Text>
+
+<Text style={styles.bio}>Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.</Text> */
+}
```
Comment (jake-pauls) [inline app/src/components/ProfileComponent/ProfileComponent.js:37]: Can this be removed? Or is this template code for something? Just thinking we might want to keep the code base clean if this isn't being documented for a special reason :smile:

File: app/src/components/ProfileComponent/styles.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      fontWeight:'bold',
+      fontSize: 20,
+  },
+
+  bio: {
+    marginTop: 20,
+    width: "85%",
+    textAlign: "center",
+  },
+});
```
Comment (jake-pauls) [inline app/src/components/ProfileComponent/styles.js:69]: I see, now you know why I like styled components and component libraries :laughing: 

Also, if we don't need the extra comments, we might want to keep it clean.

File: app/src/components/auth/LoginButton/styles.js [hunk 3]
```diff
@@ -2,10 +2,10 @@ import { StyleSheet } from "react-native";
 
 const styles = StyleSheet.create({
   button: {
+    // backgroundColor: "#E43090",
```
Comment (jake-pauls) [inline app/src/components/auth/LoginButton/styles.js:5]: More of the comment thing, up to you

**Review by jake-pauls — APPROVED** [review]
Overall looks good, just some code quality stuff (the leftover comments) might want to be cleaned up. I kind of want to get a Prettier config in this repo so that we can have consistent code style as well... let me know what you think. I just see extra whitespace and differently organized imports and Prettier can solve that. If we want to hangout with this repo for a while it'd be nice.

File: app/src/components/ProfileComponent/ProfileComponent.js (hunk 1 above)
Comment (AlexLazcano) [reply app/src/components/ProfileComponent/ProfileComponent.js:37]: These changes were from a month ago so I don't remember why it commented out 😅. But I still need to change this component to a styled component so I will clean it up when I do

**Comment by AlexLazcano** [comment]
@jake-pauls I meant to merge this to develop lol. So I will change that. As for prettier, yeah that sounds good! I can also install husky so that it auto does it when we commit. Might be good to use prettier and [standard js](https://standardjs.com/). Standard js make it a lot cleaner like it won't let you commit if there are unused imports and stuff like that. Let me know what you think

**Comment by jake-pauls** [comment]
> @jake-pauls I meant to merge this to develop lol. So I will change that. As for prettier, yeah that sounds good! I can also install husky so that it auto does it when we commit. Might be good to use prettier and [standard js](https://standardjs.com/). Standard js make it a lot cleaner like it won't let you commit if there are unused imports and stuff like that. Let me know what you think

Sounds great. Up to you about merging into development or main! Like I mentioned in the other PR the branching strategy is totally up to you.

**Comment by AlexLazcano** [comment]
> > @jake-pauls I meant to merge this to develop lol. So I will change that. As for prettier, yeah that sounds good! I can also install husky so that it auto does it when we commit. Might be good to use prettier and [standard js](https://standardjs.com/). Standard js make it a lot cleaner like it won't let you commit if there are unused imports and stuff like that. Let me know what you think
> 
> Sounds great. Up to you about merging into development or main! Like I mentioned in the other PR the branching strategy is totally up to you.

How about we merge what we have into develop, then merge develop into main, then we can start branching off from that. Also, thoughts on releasing before merging whenever we merge changes?

**Comment by jake-pauls** [comment]
> > > @jake-pauls I meant to merge this to develop lol. So I will change that. As for prettier, yeah that sounds good! I can also install husky so that it auto does it when we commit. Might be good to use prettier and [standard js](https://standardjs.com/). Standard js make it a lot cleaner like it won't let you commit if there are unused imports and stuff like that. Let me know what you think
> 
> > 
> 
> > Sounds great. Up to you about merging into development or main! Like I mentioned in the other PR the branching strategy is totally up to you.
> 
> 
> 
> How about we merge what we have into develop, then merge develop into main, then we can start branching off from that. Also, thoughts on releasing before merging whenever we merge changes?

Sure, if you merge this into development it should update my outgoing PR. Not sure I understand your other question. Feel free to DM me about it if it's easier. 🙂

**Pull request merged**